primary/src/block_synchronizer/responses.rs

```diff
@@ -0,0 +1,133 @@
+use blake2::digest::Update;
+use config::Committee;
+use crypto::{traits::VerifyingKey, Digest, Hash};
```

Comment:
I believe this is the file tripping up the license checker. I don't know why the script is no longer printing, perhaps because it is run in an environment which captures its stdout now that "check licenses" is a separate job?